modules/matterhorn-admin-ui-ng/src/main/webapp/scripts/shared/partials/wizards/new-event.html

```diff
@@ -413,7 +413,10 @@
                         data-disable-search-threshold="8"
                         data-search_contains="true"
                         ng-model="wizard.step.ud.workflow"
-                        ng-options="w.title for (obj, w) in wizard.step.workflows"
+                        ng-options="w.title disable when 
```

Comment:
Why would you want to display the workflows in the drop-down list if the user cannot select them? Can't they just be omitted in the list?

Reply:
There is no inbuilt functionality in angular to omit them. It can be done with a filter if required but I was trying to make the fewest changes to the code. (see ticket)

Reply:
... the filter seems not that hard to implement. If you don't need them "greyed", could you do the filter?